resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
```

Comment:
We have Babel in the project precisely so that we can use ES6 features, so doing so would be consistent with the other usage in the repo.